redaction/draft-strad-trans-redaction.md

```diff
@@ -69,26 +75,133 @@ certificates.
 TODO: Highlight better the differences between registered domains and
 subdomains, referencing the relevant DNS RFCs.
 
-Section TBD of [I-D.ietf-trans-rfc6962-bis] proposes two mechanisms for dealing
-with this conundrum: wildcard certificates and name-constrained intermediate
-CAs. However, these mechanisms are insufficient to cover all use cases.
-
-TODO(eranm): Expand on when each of the other mechanisms is suitable and when
-this mechanism may be suitable.
-
-We define a domain label redaction mechanism that covers all use cases, at the
-cost of increased implementation complexity. CAs and domain owners should note
-that there are privacy considerations ({{privacy_considerations}}) and that
-TLS clients may apply additional requirements (relating to the use of this
-redaction mechanism) for a certificate to be considered compliant.
-
 # Requirements Language
 
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
 "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be
 interpreted as described in [RFC2119].
 
-# Redacting Labels in Precertificates    {#redacting_labels}
+# Redaction Mechanisms
+
+We propose three mechanisms to deal with this conundrum, in increasing order of
```

Comment:
Good point.  Addressed by https://github.com/robstradling/certificate-transparency-rfcs/commit/8ffb8cf7b7bc0cd94867f15fffb96d6f4d1b4c2f (by repeating/summarizing the first half of the Abstract).